CONTRIBUTING.md

```diff
@@ -155,6 +155,34 @@ We use [Milestones](https://github.com/bevyengine/bevy/milestones) to track issu
 
 There are also two priority labels: [`P-Critical`](https://github.com/bevyengine/bevy/issues?q=is%3Aopen+is%3Aissue+label%3AP-Critical) and [`P-High`](https://github.com/bevyengine/bevy/issues?q=is%3Aopen+is%3Aissue+label%3AP-High) that can be used to find issues and PRs that need to be resolved urgently.
 
+### Closing PRs and Issues
+
+From time to time, PRs are unsuitable to be merged in a way that cannot be readily fixed.
+Rather than leaving these PRs open in limbo indefinitely, they should simply be closed.
+
+This might happen if:
+
+1. The PR is spam or malicious.
+2. The work has already been done elsewhere or is otherwise fully obsolete.
+3. The work is particularly low quality, and the author is resistant to coaching.
+4. The work adds features or abstraction of limited value, especially in a way that could easily be recreated outside of the engine.
+5. The work has been sitting in review for so long and accumulated so many conflicts that it would be simpler to redo it from scratch.
+6. The PR is pointlessly large, and should be broken into multiple smaller PRs for easier review.
```

Comment:
An edge case would be large PRs that are left open while the small PRs are made, but I guess we should probably also just close them. Closed PRs can still be linked to from the other smaller PRs

Reply:
Yeah, I think that's generally the way.